apps/walkthrough/snapshot_origins_cookiecutter/{{cookiecutter.namespace}}/{{cookiecutter.snapshot_version}}/{{cookiecutter.short_name}}.{{cookiecutter.file_extension}}.dvc

```diff
@@ -1,22 +1,22 @@
 meta:
   origin:
-    dataset_title_owid: {{cookiecutter.dataset_title_owid}}
-    {%- if cookiecutter.dataset_title_producer %}
-    dataset_title_producer: {{cookiecutter.dataset_title_producer}}
+    title_snapshot: {{cookiecutter.title_snapshot}}
+    {%- if cookiecutter.title %}
```

Comment:
I think what we decided is that `title` should be mandatory, not `title_snapshot`. Usually, we'll just give the former. But in those cases where data product and snapshot differ, then we'll specify `title_snapshot`. Does that involve additional logic elsewhere?

Reply:
Oh sorry, this is walkthrough and I haven't looked into it much as it is going to be removed right after merging this PR. (I agree with what you say)